app/views/cars/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Cars</h1>
```

Comment:
Here, maybe add comments to help with what we shall do in this view and how we are going to display the cars.